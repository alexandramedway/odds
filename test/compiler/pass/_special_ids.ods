do PI
do EUL

do print(PI)
do print(EUL)
