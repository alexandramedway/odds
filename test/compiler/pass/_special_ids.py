3.14159265
2.7182818
print(3.14159265)
print(2.7182818)
