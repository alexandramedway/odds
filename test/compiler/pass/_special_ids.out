3.14159265
2.7182818
